utility/src/main/java/org/oppia/android/util/parser/html/HtmlParser.kt

```diff
@@ -128,6 +131,15 @@ class HtmlParser private constructor(
       htmlContent, imageGetter, computeCustomTagHandlers(supportsConceptCards, htmlContentTextView)
     )
 
+    val websiteLink = "https://www.oppia.org"
+    val start = htmlSpannable.indexOf(websiteLink)
+    val end = start + websiteLink.length
+    if (start != -1) {
+      val urlSpan = URLSpan(websiteLink)
+      htmlSpannable.setSpan(
+        urlSpan, start, end, Spanned.SPAN_EXCLUSIVE_INCLUSIVE
+      )
+    }
```

Comment:
This is the only code that actually needs to be added to this PR to fix the issue. I have a suggestion on how to improve it so that the fix applies to all links, not just the Oppia website. 
```suggestion
    val urlPattern = Patterns.WEB_URL
    val matcher = urlPattern.matcher(htmlSpannable)

    while (matcher.find()) {
      val start = matcher.start()
      val end = matcher.end()
      val url = htmlSpannable.subSequence(start, end).toString()

      val urlSpan = URLSpan(url)
      htmlSpannable.setSpan(urlSpan, start, end, Spanned.SPAN_EXCLUSIVE_INCLUSIVE)
    }
```